prow/cluster/cert-manager/cluster-issuer.yaml

```diff
@@ -17,3 +17,28 @@ spec:
       name: letsencrypt-private-key
     # Enable the HTTP-01 challenge provider
     http01: {}
+---
+# cluster-issuer
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  namespace: default
```

Comment:
remove this, clusterIssuer is cluster wide.

Reply:
Done